src/components/store/PickupEventDropdown/style.module.scss

```diff
@@ -0,0 +1,42 @@
+@use 'src/styles/vars.scss' as vars;
+
+.eventPicker {
+  display: flex;
+  flex-direction: column;
+  min-height: 400px;
```

Comment:
Why do we have min-height here? I think it makes the spacing a little weird
<img width="1440" alt="Screenshot 2024-08-13 at 11 26 34 PM" src="https://github.com/user-attachments/assets/da49b521-bf1d-4ebe-93a0-9d221260a4d6">